egui_plot/src/legend.rs

```diff
@@ -86,6 +99,23 @@ impl Legend {
         self.hidden_items = Some(hidden_items.into_iter().collect());
         self
     }
+
+    /// Specifies if the legend item order should be the inserted order.
```

Comment:
as opposed to what?
maybe this should also be an `enum`?

Reply:
The legends currently get sorted due to the b-tree.

Reply:
maybe add another doc comment here